src/main/java/srver/vulnerability.java

```diff
@@ -0,0 +1,158 @@
+package srver;
+
+/**
+ * Created by Ксения on 12.12.2016.
+ */
+
+/**
+ * Created by Ксения on 25.11.2016.
+ */
+public class vulnerability {
+    String lastseen = null;
+    String references = null;
```

Comment:
&#x1F34E; **Issue**
You don't need to initialize field with null explicitly, its initialized to null by default in java.